src/components/AppHeader/LoginButton/LoginButton.vue

```diff
@@ -1,14 +1,14 @@
 <template>
-    <button class="loginButton"
-            type="button">
+    <router-link class="loginButton"
+            to="/login">
       <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
Forgot it

Reply:
I corrected this (and the logo too) by using the same method as the add icon button. I don't think we can apply the same method as in the icon component, because of the loop.

Reply:
@KamiAmiga  yes, I see we've added the `<li>`  tag in `Icon.vue`  and it doesn't fit our need. I think we need the Icon component to manage both possible usage.
@frisouilles  can you have a look at it ?  Maybe create a parent component something like 
IconItem.vue that will take care of the `<li>` so we can use only Icon.vue as a stand alone for our other specific cases.
thx